features/jsonld/context.feature

```diff
@@ -41,7 +41,6 @@ Feature: JSON-LD contexts generation
             "jsonData": "Dummy/jsonData",
             "arrayData": "Dummy/arrayData",
             "nameConverted": "Dummy/nameConverted",
-            "id": "Dummy/id",
```

Comment:
Why do we need to change this @soyuka? It looks like a BC break.

Reply:
I've done this change. It's because of you :stuck_out_tongue:
https://github.com/symfony/symfony/pull/31486

Reply:
That's also why I've added what you've done for MongoDB too: https://github.com/api-platform/core/pull/2784/files#diff-c5fed7f9ec0eb28b0039798760fac3ac

Reply:
Got it!